lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -167,11 +223,10 @@ export class ScopeSet {
      * Returns true if the set of original scopes only contained client-id
      */
     isLoginScopeSet(): boolean {
-        const hasLoginScopes = (
-            this.originalScopes.has(this.clientId) ||
-            this.originalScopes.has(Constants.OPENID_SCOPE) ||
-            this.originalScopes.has(Constants.PROFILE_SCOPE)
-        );
+        const hasLoginScopes =
+                   this.originalScopes.has(this.clientId) ||
+                   this.originalScopes.has(Constants.OPENID_SCOPE) ||
+                   this.originalScopes.has(Constants.PROFILE_SCOPE);
```

Comment:
nitpick but removing parentheses lessens readability IMO

Reply:
I did not change this, original code had it. But can remove the parantheses.